use configuration in the database (#109)
mattermost/mattermost-helm: charts/mattermost-enterprise-edition/README.md

$ helm inspect values mattermost/mattermost-push-proxy
```

# 4. Upgrading to Mattermost Chart Version 1.0.0

> This Chart version uses the configuration in the database and environment variables for non changing configuration.

Reviewer: ```suggestion
This chart version uses the configuration in the database and environment variables for non-changing configuration.
```

Author: done

> To upgrade from previous Helm Chart version to 1.0.0 you need to do some steps before applying the new chart

Reviewer: ```suggestion
To upgrade from a previous chart version to 1.0.0 you need to follow theses steps before applying the new chart:
```

Author: done
